### ISSUE TITLE
A11y_SQLdatabase projects extension_View And More Action_Update Project From Database_Voiceover: Label names are not being announced by voiceover.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#22099]
**NOTE: "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**
### GitHub Tags:
 #A11ySev1; #A11yWCAG2.1; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #Name Role Value; #WCAG4.1.2 #MAC; #AINotInScope; #Voiceover; 

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3
### Repro Steps:

1. Start Voiceover with using shortcut key Command+F5.
2. Launch Azure Data Studio
3. Tab till database project and press enter.
4. “Database Project” panel will open.
5. Tab till “View And More Action” button and press enter.
6. Tab till “Update Project From Database” and press enter.
7. Verify whether associated label names are being announced by voiceover or not.

### Actual Result:​
Voiceover is not announcing associated label names for 'Server, database and location' controls.

### Expected Result:​
Voiceover should be announcing associated label names for 'Server, database and location' controls.
Also, voiceover should be announce required for required fields.

### User Impact:
Voiceover users will face difficulty while navigating if voiceover is not announcing label name for the 'Server, database and location' controls. Hence users might not be able to understand the functionality and the purpose of the controls.

### Attachment:
![Label Name not being announce](https://user-images.githubusercontent.com/86241494/222366379-fbf70a58-3f98-4fa6-8c92-73855e57b4af.gif)

**Comment by kisantia** [comment]
@SachinChavhan317 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by kisantia** [comment]
@SachinChavhan317 please verify this is fixed. Thanks!

**Comment by kisantia** [comment]
@SachinChavhan317 please verify this issue is fixed when you have a chance. Thanks!

**Comment by manaswita-chougule** [comment]
#Closed; #Regressed:04-11-23;
Verified the issue in below environment
SQL Azure Data Studio : Version 1.44.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is fixed, hence closing the bug. Please find below attachment for reference. Thanks!
![22099_fixed](https://user-images.githubusercontent.com/91586101/231139619-d2783237-ef80-4599-8753-67d715e4b95f.gif)
